pkg/commands/loading_commits.go

```diff
@@ -312,14 +312,23 @@ func (c *CommitListBuilder) setCommitMergedStatuses(refName string, commits []*m
 }
 
 func (c *CommitListBuilder) getMergeBase(refName string) (string, error) {
+	baseBranch := c.GitCommand.TryGetMainBranch()
```

Comment:
I would benchmark how long it takes to run this function, because we could save some time by only running it once and storing the result on the git struct

Reply:
There are multiple structs that have something to do with git, i hope i picked a good one :)